galaxy_ng/app/models/container.py

```diff
@@ -16,6 +16,16 @@ class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
 
 
+class ContainerNamespace(
+        container_models.ContainerNamespace,
```

Comment:
Mild warning about subclassing classes and using the same name... It can get confusing sometimes. Especially for code that does typechecking `isinstance(foo, ContainerNamespace)` etc. Can be confusing to read.

If it works and or is required to work, leave it and don't worry about it.

Reply:
Didn't notice this was a proxy model. Not sure if that confuses me less or more. But if that is the idiomatic approach, works for me.

Reply:
This is a good point. We should have done this with all our proxy models. I'm not sure what the right thing to do is

Reply:
I think I'll leave it as is, since we aren't renaming any of the other proxy models, and I think renaming them will create more migrations